### PR TITLE
docs(zh-CN): refactor machine translation

### PR DESCRIPTION
**Pull request by zzy987**
## Description

Remove machine translations.

## Related Issue
#781 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jim3ma** [comment]
Discard the change of `deploy/helm-charts`.

**Review by 244372610 — APPROVED** [review]

File: docs/zh-CN/cli-reference/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # CLI Reference
 
-对于几乎所有用户，命令行是您可能需要的第一个参考。 [CLI Reference](cli-reference)目录下的文档是关于Dragonfly CLI的命令详细使用，包括`dfget`、`cdn`、`scheduler`和`manager`。您可以获得有关命令的介绍、概要、示例和选项。最后但同样重要的是，Dragonfly 可以保证命令行文档与 Dragonfly CLI 的源代码高度一致。更重要的是，所有命令行文档都是通过源代码自动生成的。
+如何使用命令行来启动 dragonfly 可能是您遇到的第一个问题。 [CLI Reference](cli-reference)目录下的文档介绍了使用命令行启动 Dragonfly 的发方法，包括`dfget`、`cdn`、`scheduler`和`manager`等子系统的启动。您可以找到命令的简介、示例和可选参数。
```
Comment (yxxhero) [inline docs/zh-CN/cli-reference/README.md:3]: 您遇到的第一个问题可能是如何使用命令行来启动 dragonfly

File: docs/zh-CN/cli-reference/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+
+所有命令行可选参数都是通过由源代码自动生成的。
```
Comment (yxxhero) [inline docs/zh-CN/cli-reference/README.md:6]: 通过和由保留一个

File: docs/zh-CN/cli-reference/dfget.md [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
 
 ```
 dfget --schedulers 127.0.0.1:8002 -o /path/to/output -u "http://example.com/object"
 ```
 
-## Log configuration
+## 查看日志
 
-如果您想要在Terminal中查看日志，请使用 --console 将环境变量 console 设置为 true。
+如果您想要在 Terminal 中查看日志，请使用 --console 将环境变量 console 设置为 true。
```
Comment (yxxhero) [inline docs/zh-CN/cli-reference/dfget.md:25]: 请使用 --console参数，或者将环境变量 console 设置为 true。

File: docs/zh-CN/cli-reference/dfget.md [hunk 4]
```diff
@@ -61,11 +63,11 @@ dfget --schedulers 127.0.0.1:8002 -o /path/to/output -u "http://example.com/obje
 
 # dfget daemon
 
-## Log configuration
+## 查看日志
 
 如果您想要在Terminal中查看日志，请使用 --console 将环境变量 console 设置为 true。
```
Comment (yxxhero) [inline docs/zh-CN/cli-reference/dfget.md:69]: see above.

**Comment by yxxhero** [comment]
need rebase @zzy987

**Comment by yxxhero** [comment]
@zzy987 Please fix conflicts. Thanks very much.

**Review by jim3ma — APPROVED** [review]

**Review by jim3ma — CHANGES_REQUESTED** [review]
Update helm-charts

**Review by yxxhero — DISMISSED** [review]

**Comment by yxxhero** [comment]
@zzy987 Please fix markdown lint issue. Thanks very much.

**Comment by yxxhero** [comment]
@zzy987 Thanks very  much agagin. Thank for your work.

**Review by yxxhero — COMMENTED** [review]
LGTM

**Review by yxxhero — APPROVED** [review]

**Comment by yxxhero** [comment]
@zzy987 you should mark the request changes to be resolved. So I can merged it.

**Pull request merged**